frontend/templates/troubleshooting/index.tsx

```diff
@@ -57,9 +57,23 @@ import { HeadingSelfLink } from './components/HeadingSelfLink';
 import ProblemCard from './Problem';
 import { PixelPing } from '@components/analytics/PixelPing';
 import { TagManager, GoogleNoScript } from './components/TagManager';
+import { ScrollPercent } from './scrollPercent';
+import { LinkToTOC, TOCContextProvider } from './tocContext';
+import {
+   TOC,
+   onlyShowIfTOCFlagEnabled,
+   onlyShowIfTOCFlagEnabledProvider,
+} from './toc';
 import { ViewStats } from '@components/common/ViewStats';
 import { IntlDate } from '@components/ui/IntlDate';
 
+const RelatedProblemsTitle = 'Related Problems';
+
+const FlaggedTOC = onlyShowIfTOCFlagEnabled(TOC);
+const FlaggedScrollPercent = onlyShowIfTOCFlagEnabled(ScrollPercent);
+const FlaggedTOCContextProvider =
+   onlyShowIfTOCFlagEnabledProvider(TOCContextProvider);
```

Comment:
This is the beating heart of the stubbing right here? Disabled in production.

Reply:
Yep!